src/lib/apparmor/profiles.rb

```diff
@@ -107,6 +107,19 @@ def all
     def toggle(name)
       @prof[name].toggle
     end
+
+  private
+
+    # Returns the output of the given command
+    #
+    # @param args [Array<String>] the command to execute (first element of the array)
+    #   and its arguments (remaining elements).
```

Comment:
in fact cheetah allows even more fancy args like `["ls"], ["grep", "."]` which pipes command together.
Lets try it yourself:
```
irb(main):004:0> Cheetah.run(["ls", "/tmp"], ["grep", "backticks"], stdout: :capture)
=> "backticks.log\n"
```

Reply:
Yes, I saw it in the documentation, but I was not sure if it deserves to document both possible `args`, I mean, Array<String> and Array<Array<String>>. What do you think? Maybe with a simple reference to Cheetah.

Reply:
so maybe just point to Cheetah with possible arguments?